src/components/structured-text-block/structured-text-block.vue

```diff
@@ -149,6 +149,25 @@
         })
       }
       case 'StructuredTextImageRecord': {
+        let sizes;
+
+        switch (record.layout) {
+          case 'left':
+          case 'right': {
+            sizes = '(min-width: 1100px) 430px, (min-width: 720px) 40vw, 90vw'
+            break
+          }
+          case 'narrow': {
+            sizes = '(min-width: 1100px) 680px, (min-width: 720px) 60vw, 90vw'
+            break
+          }
+          case 'default':
+          default: {
+            sizes = '(min-width: 1100px) 860px, (min-width: 720px) 75vw, 90vw'
+            break
+          }
+        }
+
```

Comment:
Code works & is clear so feel free to merge but I have a nitpick, I think an object mapping would improve readability (even so left & right have the same size):
```
const layoutSizesMapping = { left: '', right: '', ... };

image: { sizes: layoutSizesMapping[record.layout] }
```